data/remnant/remnant news.txt

```diff
@@ -303,3 +305,77 @@ news "remnant cafeteria"
 			"are examining holo projections over a large meal"
 		word
 			"."
+
+
+
+news "remnant comprehensible"
+	location
+		government "Remnant"
+	to show
+		has "remnant: sign studies complete"
+	name
+		word
+			"Spaceport"
+			"Loading dock"
+			"Cafeteria"
+			"Landing pad"
+			"Landing dock"
+			"Spaceport alley"
+			"Spaceport hallway"
+	message
+		phrase
+			"Amount of Remnant"
+		word
+			" sign "
+			" gesture "
+		word
+			"quickly"
+			"calmly"
+			"leisurely"
+			"slowly"
+			"deliberately"
+			"carefully"
+			"precisely"
+			"sharply"
+			"briskly"
+			"sweepingly"
+			"intensively"
+			"meticulously"
+			"at length"
+			"vaguely"
+			"briskly"
+			"hurriedly"
+			"hastily"
+			"abruptly"
+			"spontaneously"
+		word
+			" to "
+		word
+			"each other"
+			"another Remnant"
+			"other Remnant"
+			"a Remnant at the information desk"
+			"a dockworker"
+			"something or someone you can't see"
+		word
+			". "
+		word
+			"From this angle, "
+			"Without them facing you, "
+		word
+			"all you can "
+			"what you can "
+		word
+			"make out"
+			"catch"
+			"see"
+			"discern"
+		word
+			" is that they are "
+		word
+			"discussing a recent salvage operation"
+			"making lunch plans"
+			"arguing about engine efficiencies"
+			"gossipping about a visitor from outside the Ember Waste"
```

Comment:
```suggestion
			"gossiping about a visitor from outside the Ember Waste"
```

Reply:
While both "gossipping" and "gossiping" are acceptable spellings, it seems the former is in fewer spellcheck dictionaries. Thanks for the correction. :-)